lib/index.js

```diff
@@ -317,6 +317,7 @@ function writeRequireJS (version, destination, minify, callback) {
             if (base === 'Gruntfile') { return; }
             var relativePath = path.relative(famousPath, path.dirname(file));
             var module = path.join(namespace, relativePath, base);
+            module = module.split('\\').join('/');
```

Comment:
I think this is the right thing to do.. you will not end up getting anything splitting in a module name not coming from a windows file system